packages/blade/src/components/Icons/Icons.stories.tsx

```diff
@@ -13,7 +13,7 @@ const Page = (): ReactElement => {
     <StoryPageWrapper
       componentDescription="We provide a bunch of icons out-of-the-box for Blade in 6 different sizes. You can choose which size & color fits the best for your use case using the color & size props."
```

Comment:
```suggestion
      componentDescription="Blade provides a bunch of icons out of the box in 6 different sizes. You can choose the size & color that fits best for your use case using the color & size props."
```

Reply:
Done in https://github.com/razorpay/blade/pull/883/commits/6bd812011991d0077ccfdbc28db1625c68607145